playbooks/roles/ocp-install/templates/ocp_install_scripts_create_admin_user.j2

```diff
@@ -0,0 +1,5 @@
+# {{ ansible_managed }}
+
+oc create secret generic htpass-secret --from-file=htpasswd={{ workdir }}/htpasswd -n openshift-config
+oc apply -f {{ workdir }}/ocp_oauth.yaml
+oc adm policy add-cluster-role-to-user cluster-admin admin
```

Comment:
Maybe we can have this as ansible task itself instead of creating a shell file.

Reply:
Sure. But this is in line with other tasks in the playbook.